### PR TITLE
web3: added Utils methods to static Web3 class

### LINKED ISSUE #27851: [@types/web3] Add static Utils property to Web3 class
Hi all,

As I know, utility functions are also exposed on the Web3 class as well as on instance of Web3. But I've found Utils only in an instance.

Compare this to the original docs http://web3js.readthedocs.io/en/1.0/web3.html#utils

What is missing:
- static Web3.utils property;

- [x] I tried using the `@types/web3` package and had problems.
- [x] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @simon-jentzsch, @nitzantomer, @zurbo, @yxliang01, @phra, @naddison36, @icaroharry, @linusnorton, @jpeletier, @anneau, @matrushka, @andrevmatos, @levino, @zlumer

### PR DESCRIPTION
**Pull request by zlumer**
Fixes #27851 

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: http://web3js.readthedocs.io/en/1.0/web3.html#utils

### CODE REVIEW

**Comment by archangel-irk** [comment]
I think the current state of PR is not fixing the issue, because of the 'utils' is a static property of Web3 class.

It could be resolved like this:
```js
declare class Web3 {
  ...
  static utils: Utils;
  ...
```

File: types/web3/web3-tests.ts [hunk 1]
```diff
@@ -20,3 +20,4 @@ myContract.options.gas = 5000000;
 
 const weiStr: string = web3.utils.toWei("100", "gwei");
 const weiBn: BigNumber = web3.utils.toWei(web3.utils.toBN("1"));
+const rndHex: string = Web3.randomHex(10);
```
Comment (Levino) [inline types/web3/web3-tests.ts]: `Web3` does not have a method `randomHex`. `Web3.utils` has.

**Review by levino — CHANGES_REQUESTED** [review]
This PR would introduce a bug instead of fixing the issue. Maybe the issue should be made more precise for collaborators to understand the problem. https://github.com/DefinitelyTyped/DefinitelyTyped/issues/27851

**Comment by zlumer** [comment]
@Levino good catch, fixed.

**Comment by zlumer** [comment]
See also #27963

**Comment by Levino** [comment]
@zlumer @archangel-irk was faster than you. Lets close your PR in favour of https://github.com/DefinitelyTyped/DefinitelyTyped/pull/27963

Remark: If we would accept your PR we would have f42b3db14b81cf756a2f6da5c61f341c315f85d1 forever in the history (including a bug). If anyone would be stupid enough to cherry pick, they would have a problem. It is better to remove such footguns from the history by rebasing and squashing the commits into one and then force push over your branch here. This is just a remark for future PRs.

**Comment by zlumer** [comment]
@Levino agree, closing.

**Pull request closed without merging**